yarn-project/merkle-tree/src/index.ts

```diff
@@ -6,7 +6,7 @@ export * from './pedersen.js';
 export * from './sparse_tree/sparse_tree.js';
 export { StandardIndexedTree } from './standard_indexed_tree/standard_indexed_tree.js';
 export * from './standard_tree/standard_tree.js';
-export { INITIAL_LEAF } from './tree_base.js';
+export { INITIAL_LEAF, getTreeMeta } from './tree_base.js';
```

Comment:
Is this a leftover? Not seeing it being used 👀

Reply:
It's not. I use it [here](https://github.com/AztecProtocol/aztec-packages/blob/c0046ea17c9e000c55fa63fcba1eb6e14fae1a9b/yarn-project/world-state/src/world-state-db/merkle_trees.ts#L568) to determine whether the DB is populated.